apps/settings/js/panels/simpin/simpin.js

```diff
@@ -34,12 +36,11 @@ define(function(require) {
       return Sanitizer.escapeHTML `
         <li class="simpin-enabled simpin-enabled-${simIndex}
           simpin-${simIndex}">
-          <label class="pack-switch">
-            <input type="checkbox" data-ignore data-sim-index="${simIndex}"
-              data-type="checkSimPin"/>
-            <span data-l10n-id="simPinWithIndex"
-              data-l10n-args="${simPinArgs}"></span>
-        </label>
+          <gaia-switch data-ignore data-sim-index="${simIndex}"
+              data-type="checkSimPin">
```

Comment:
therefore we don't need `data-type="checkSimPin"` here sinice we don't go through handleEvent